cmd/core-command/Dockerfile

```diff
@@ -49,5 +52,10 @@ WORKDIR /
 COPY --from=builder /edgex-go/cmd/core-command/Attribution.txt /
 COPY --from=builder /edgex-go/cmd/core-command/core-command /
 COPY --from=builder /edgex-go/cmd/core-command/res/configuration.toml /res/configuration.toml
+
+# Copy over /tmp/passwd file with injected "user" entry as required by Docker's "USER <user>" command
+COPY --from=builder /tmp/passwd /etc/passwd
+USER edgex_user
```

Comment:
I would like to try doing this from compose file, rather than the Dockerfiles.
```
user: "${EDGEX_USER}:${EDGEX_GROUP}"
```
define EDGEX_USER and EDGEX_GROUP in .env file and use them through out compose and entry point scripts.
```
EDGEX_USER=2002
EDGEX_GROUP=2001
```

Reply:
IMO, for image itself, i think it makes sense to define it as ENV variable for docker run to use; doesn't have to be in compose-builder only but also any docker run command